tests/test_endpoints.py

```diff
@@ -71,6 +72,18 @@ def test_base(self):
         self.expected_request_headers['x-myobapi-cftoken'] = ''
         self.assertEndpointReached(self.myob.info, {}, 'GET', '/Info/')
 
+    @patch('myob.managers.requests.request')
+    def test_empty_delete_response(self, mock_request):
+        mock_request.return_value.status_code = 200
+
+        def response_json():
+            return Response().json()
```

Comment:
Better to explicitly be raising a `JSONDecodeError` error here -- that's what we want to be testing for.